cves/CVE-2011-3923.yml

```diff
@@ -111,10 +100,13 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
-
+  answer: |
+    The fixing commit for the vulnerability shows 32 insertions for a
```

Comment:
"The fixing commit" reads a little odd, perhaps "The fix that was committed for the vulnerability" might work better here.

Reply:
I totally agree. I'll fix this in the next commit.